lib/android-tools-info.ts

```diff
@@ -284,8 +282,8 @@ export class AndroidToolsInfo implements IAndroidToolsInfo {
 		return buildToolsVersion;
 	}
 
-	private async getAppCompatRange(): Promise<string> {
-		let compileSdkVersion = await this.getCompileSdk();
+	private getAppCompatRange(): string {
+		let compileSdkVersion = this.getCompileSdk();
```

Comment:
why not name the "getCompileSdk()" method as "getCompileSdkVersion"? It returns a number and it's assigned to a variable sdkVersion, therefore I think it would be more correctly named this way